cmd/check_container.go

```diff
@@ -99,13 +118,89 @@ var checkContainerCmd = &cobra.Command{
 			return err
 		}
 
+		// submitting results to pxysis if submit flag is set
+		if s, _ := cmd.Flags().GetBool("submit"); s {
+			log.Info("preparing results that will be submitted to Red Hat")
+
+			certImageJsonFile, err := os.Open(path.Join(artifacts.Path(), certification.DefaultCertImageFilename))
+			defer certImageJsonFile.Close()
+			if err != nil {
+				return err
+			}
+
+			certImageBytes, err := io.ReadAll(certImageJsonFile)
+			if err != nil {
+				return err
+			}
+
+			certImage := new(pyxis.CertImage)
+			err = json.Unmarshal(certImageBytes, &certImage)
+			if err != nil {
+				return err
+			}
+
+			rpmManifestJsonFile, err := os.Open(path.Join(artifacts.Path(), certification.DefaultRPMManifestFilename))
+			defer rpmManifestJsonFile.Close()
+			if err != nil {
+				return err
+			}
+
+			rpmManifestBytes, err := io.ReadAll(rpmManifestJsonFile)
+			if err != nil {
+				return err
+			}
+
+			rpmManifest := new(pyxis.RPMManifest)
+			err = json.Unmarshal(rpmManifestBytes, rpmManifest)
+			if err != nil {
+				return err
+			}
+
+			testResultsJsonFile, err := os.Open(path.Join(artifacts.Path(), certification.DefaultTestResultsFilename))
+			defer testResultsJsonFile.Close()
+			if err != nil {
+				return err
+			}
+
+			testResultsBytes, err := io.ReadAll(testResultsJsonFile)
+			if err != nil {
+				return err
+			}
+
+			var testResults = new(pyxis.TestResults)
+			err = json.Unmarshal(testResultsBytes, &testResults)
+			if err != nil {
+				return err
+			}
+
+			pyxisAPIToken, _ := cmd.Flags().GetString("pyxis_api_token")
+			projectID, _ := cmd.Flags().GetString("pyxis-project-id")
+
+			pyxisEngine := pyxis.NewPyxisEngine(pyxisAPIToken, projectID, &http.Client{Timeout: 60 * time.Second})
+
+			//TODO: use the return values once we know what we need to display to the user
+			_, _, _, err = pyxisEngine.SubmitResults(certImage, rpmManifest, testResults)
+			if err != nil {
+				return err
+			}
+		}
+
 		return nil
 	},
 }
 
 func init() {
+	checkContainerCmd.Flags().BoolP("submit", "s", false, "submit check container results to red hat")
+	viper.BindPFlag("submit", checkContainerCmd.Flags().Lookup("submit"))
+
+	checkContainerCmd.Flags().StringP("docker-config", "d", "", "path to docker config.json file")
+	viper.BindPFlag("docker_config", checkContainerCmd.Flags().Lookup("docker-config"))
+
 	checkContainerCmd.Flags().String("pyxis-api-token", "", "API token for Pyxis authentication")
 	viper.BindPFlag("pyxis_api_token", checkContainerCmd.Flags().Lookup("pyxis-api-token"))
 
+	checkContainerCmd.Flags().String("pyxis-project-id", "", "Project ID for a Pyxis container project")
```

Comment:
I thought this was going to be certification-project-id?

Reply:
It is, I had yet to rebase.